### PR TITLE
Update .gitignore file

### PR DESCRIPTION
**Pull request by ethanschreur**
**What does this PR do?**
- This pull request keeps the __pycache__ folder out of the repository

**Description of the task to be completed** 
- Add `__pycache__/` to the .gitignore file

**How should this manually be tested**
- None

### CODE REVIEW

**Review by alexkayabula — CHANGES_REQUESTED** [review]
@ethanschreur rename the PR to `Update .gitignore file` then add all the files and folders you want to ignore. Don't forget to change the description of the PR accordingly.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-venv/
\ No newline at end of file
+venv/
+**__pycache__/
```
Comment (alexkayabula) [inline .gitignore:2]: Is this the right way to ignore `pycache` folder ?. It should be `__pycache__/`

**Review by alexkayabula — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ethanschreur) [reply .gitignore:2]: The pycache folder is appearing in the tests folder after I run tests so I think if I wanted to make it more specific I would write `tests/__pycache__/`.  The ** one ignores the pycache folder in all directories. Which one do you think is better?

Comment (alexkayabula) [reply .gitignore:2]: @ethanschreur use the suggestion I have given. It is better.

**Comment by alexkayabula** [comment]
@please update the description of the PR to reflect the changes made.

**Comment by alexkayabula** [comment]
@ethanschreur whenever a PR is ready for review, please re-request a review.

**Review by alexkayabula — APPROVED** [review]
LGTM

**Pull request merged**